wordpress-mobile/WordPress-Aztec-iOS
# Copy / paste support.

Implements copy / paste support.  Improves: #75.

Also fixes the node duplication issues we were having for styles.

**Known limitations:**

Still need to add support for pasting images.
Works for: bold, underline, italic, strike.
When copy-pasting large chunks of text, the editor becomes a bit slow.  [I've created a ticket for that](https://github.com/wordpress-mobile/WordPress-Aztec-iOS/issues/97).

**Test 1:**

Evaluate the new unit tests.
Run the unit tests.

**Test 2:**

1. Launch the app.
2. Copy some content from the sample post.
3. Paste it back elsewhere.
4. Switch to HTML mode and see that the HTML is fine for the supported styles.

You can try pasting styled text from other apps too.
